dts/riscv/ite/it8xxx2-pinctrl-map.dtsi

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2022 ITE Corporation. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <dt-bindings/pinctrl/it8xxx2-pinctrl.h>
+
+&pinctrl {
+	/* UART alternate function */
+	uart1_rx_default: uart1_rx_default {
```

Comment:
suggestion: node name doesn't make explicit that routed port/pin is "PB0", maybe `uart1_rx_pb0_default` would be a better choice. What if I can route uart1 rx to pb1? I can't re-use the `uart1_rx_default` name.

Reply:
Done.